wiki/osu!supporter/en.md

```diff
@@ -127,28 +129,30 @@ The remaining osu!supporter duration, as well as total amount of contributions,
 
 ## Getting osu!supporter
 
-::: Infobox
-![](img/store-product.jpg?2 "osu!supporter product banner from the osu!store")
-:::
-
 To get osu!supporter, visit the [osu!supporter store page](https://osu.ppy.sh/store/products/supporter-tag) and adjust the supporter tag duration using the slider, or the buttons with the number of months underneath. All prices are in United States dollars (USD) and don't include possible fees of the payment system.
 
 Afterwards, click `Add to Cart` to put the supporter tag into the osu!store shopping cart. To finalise the purchase, go to the [shopping cart](https://osu.ppy.sh/store/cart) and click `Checkout`, then follow on-screen prompts.
 
 ### Gifting osu!supporter
 
-osu!supporter may also be gifted to another player by entering their username into the grey box below the user card. More people can be added by going back to the store page and repeating the process.
+::: Infobox
+![](img/profile-gift-supporter.png "Dropdown menu button on a profile page for gifting someone osu!supporter")
+:::
+
+osu!supporter may also be gifted to another player by entering their username below the user card, or by clicking `Gift supporter tag` on their profile page. More people can be added by going back to the store page or someone else's profile and repeating the process.
+
+The recipient is not notified of who gave them the tag, but when gifting osu!supporter, an optional message can be attached to be sent alongside the notification email.
 
-### Conclusion
+### Confirmation
 
 After the transaction is completed, there will be a new event in the `Recent Activity` section in the profile of both the giver and the recipient:
```

Comment:
users can also choose to hide this from their own activity in the case where they are the one who purchased it, worth a mention?  
![image](https://github.com/ppy/osu-wiki/assets/83513254/19eb8a01-e7ff-4c93-94b2-82057273aa84)

Reply:
maybe something like:

```suggestion
After the transaction is completed, there will be a new event in the `Recent Activity` section in the profile of both the giver and the recipient:[^hide-event]

[^hide-event]: The giver can choose to hide this event from their `Recent Activity` on their profile when at checkout.
```